api/v1alpha1/virtualmachineimage_conversion_test.go

```diff
@@ -15,8 +15,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/vmware-tanzu/vm-operator/api/v1alpha1"
-	nextver "github.com/vmware-tanzu/vm-operator/api/v1alpha2"
-	nextver_common "github.com/vmware-tanzu/vm-operator/api/v1alpha2/common"
+	nextver "github.com/vmware-tanzu/vm-operator/api/v1alpha3"
```

Comment:
Nit - there's still several comments mentioning `"v1a2"` in the conversion & conversion test files. Sub that with something like v1a2+, v1a3 or nextver or whatever

Reply:
Will do.